R-graphql/example.r

```diff
@@ -0,0 +1,39 @@
+library("ghql")
+library("jsonlite")
+library("httr")
+
+API_URL <- "https://api.santiment.net/graphql"
+# change with real Apikey
+API_KEY <- Sys.getenv("SANBASE_API_KEY")
```

Comment:
what's the behavior if there is no such variable?

Reply:
My only concern is that the SANbase backend will find the Apikey header (with probably empty value) and the check for valid apikey will fail

Reply:
Trie it and works. You can check this for example:
```
curl \
  -X POST \
  -H "Content-Type: application/json" \
-H "Authorization: Apikey " \
--data '{ "query": "query{dailyActiveAddresses(from:\"2018-08-01 16:00:00Z\",interval:\"1d\",slug:\"dragonchain\",to:\"2018-08-05 16:00:00Z\"){activeAddresses,datetime}}" }' \
https://api.santiment.net/graphql
```

Reply:
Yes, it works because it's skipped here: https://github.com/santiment/sanbase2/blob/master/lib/sanbase_web/graphql/context_plug.ex#L64

But the empty string passes through this logic and log messages: https://github.com/santiment/sanbase2/blob/master/lib/sanbase/auth/apikey/apikey.ex#L34